2025-01-20 15:25:33,258 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:25:33,809 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 15:25:34,744 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 15:25:34,746 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:25:34,755 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:25:34,767 INFO: 127.0.0.1 - - [20/Jan/2025 15:25:34] "GET /socket.io/?EIO=4&transport=polling&t=PI46iLC HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:25:34,786 INFO: 127.0.0.1 - - [20/Jan/2025 15:25:34] "POST /socket.io/?EIO=4&transport=polling&t=PI46iOw&sid=Pj3E7JsoLqSUeuiYAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:25:34,786 INFO: 127.0.0.1 - - [20/Jan/2025 15:25:34] "GET /socket.io/?EIO=4&transport=polling&t=PI46iOz&sid=Pj3E7JsoLqSUeuiYAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:05,020 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/processors/product_name_processor.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:05,265 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:05,823 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 15:30:06,704 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 15:30:06,705 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:06,712 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:06,725 INFO: 127.0.0.1 - - [20/Jan/2025 15:30:06] "GET /socket.io/?EIO=4&transport=polling&t=PI47kh7 HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:06,746 INFO: 127.0.0.1 - - [20/Jan/2025 15:30:06] "POST /socket.io/?EIO=4&transport=polling&t=PI47ko9&sid=_p_bxns3JzAcFufHAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:30:06,758 INFO: 127.0.0.1 - - [20/Jan/2025 15:30:06] "GET /socket.io/?EIO=4&transport=polling&t=PI47koH&sid=_p_bxns3JzAcFufHAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:30,602 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/processors/product_name_processor.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:30,813 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:31,338 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 15:31:32,270 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 15:31:32,271 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:32,279 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:32,289 INFO: 127.0.0.1 - - [20/Jan/2025 15:31:32] "GET /socket.io/?EIO=4&transport=polling&t=PI483b1 HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:32,314 INFO: 127.0.0.1 - - [20/Jan/2025 15:31:32] "POST /socket.io/?EIO=4&transport=polling&t=PI483h9&sid=FYX1FOVsgrK0J6xxAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:31:32,318 INFO: 127.0.0.1 - - [20/Jan/2025 15:31:32] "GET /socket.io/?EIO=4&transport=polling&t=PI483hE&sid=FYX1FOVsgrK0J6xxAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:55,770 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/generators/text_generator.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:56,001 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:56,576 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 15:37:57,327 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 15:37:57,328 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:57,336 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:57,344 INFO: 127.0.0.1 - - [20/Jan/2025 15:37:57] "GET /socket.io/?EIO=4&transport=polling&t=PI49XcI HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:57,373 INFO: 127.0.0.1 - - [20/Jan/2025 15:37:57] "POST /socket.io/?EIO=4&transport=polling&t=PI49Xhf&sid=eZbubL9GGBpGSSEPAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:37:57,474 INFO: 127.0.0.1 - - [20/Jan/2025 15:37:57] "GET /socket.io/?EIO=4&transport=polling&t=PI49Xhw&sid=eZbubL9GGBpGSSEPAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:06,438 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/processors/product_name_processor.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:06,630 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:07,103 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 15:38:07,730 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:144]
2025-01-20 15:38:07,731 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:07,740 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:07,749 INFO: 127.0.0.1 - - [20/Jan/2025 15:38:07] "GET /socket.io/?EIO=4&transport=polling&t=PI49aCK HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:07,766 INFO: 127.0.0.1 - - [20/Jan/2025 15:38:07] "POST /socket.io/?EIO=4&transport=polling&t=PI49aEE&sid=qMVn1MVuR5AAQRibAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:07,771 INFO: 127.0.0.1 - - [20/Jan/2025 15:38:07] "GET /socket.io/?EIO=4&transport=polling&t=PI49aEI&sid=qMVn1MVuR5AAQRibAAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:30,995 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/generators/text_generator.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 15:38:31,184 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
